release_notes/release_notes.html

```diff
@@ -1,5 +1,26 @@
 <b>ThreatStream Release Notes - Published by Splunk June 15, 2021</b>
 <br><br>
+<b>Version 3.2.8 - Released November 29, 2021</b>
```

Comment:
@cpangam Could you please confirm if we also need to include release notes for the 3 changes from the Epic ticket PAPP-20942?
The changes are from PAPP-19922, PAPP-21485, and PAPP-21096.

Reply:
Yes. That was needed. These are previous release notes and not included in the release_notes.html